### DISCUSSION TITLE
how to use BetterPlayerController with BetterPlayerPlaylist

### DISCUSSION THREAD

**Discussion by akhosu** [jhomlala/betterplayer#177] [Q&A]
Hello,

I have to control BetterPlayerPlaylist, i need to send signal to go to next video. But there is no BetterPlayerController field in BetterPlayerPlaylist class and constructors. How can I connect controller to playlist player?

**Comment by jhomlala** [comment]
You can't pass custom controller to BetterPlayer's Playlist widget. Anyway you can access controller via GlobalKey and state. See this example:
https://github.com/jhomlala/betterplayer/blob/master/example/lib/pages/playlist_page.dart

Remember: do not change data source by yourself, because this will break playlist.
